src/views/auth/Signin.vue

```diff
@@ -1,27 +1,17 @@
 <template>
-  <sbc-signin @sync-user-profile-ready="onReady()" />
+  <div />
 </template>
 
 <script lang="ts">
-// Libraries
 import { Component, Vue } from 'vue-property-decorator'
 
-// Components
-import SbcSignin from 'sbc-common-components/src/components/SbcSignin.vue'
-
-// Enums
-import { Routes } from '@/enums'
-
-@Component({
-  components: {
-    SbcSignin
-  }
-})
+@Component({})
 export default class Signin extends Vue {
-  /** Called when Keycloak session is ready. */
-  private async onReady () {
-    // navigate to dashboard
-    this.$router.push({ name: Routes.DASHBOARD })
+  created () {
+    // redirect to BC Registry login page then return to this app
+    const loginUrl = sessionStorage.getItem('REGISTRY_HOME_URL') + 'login'
+    const returnUrl = encodeURIComponent(sessionStorage.getItem('BASE_URL'))
+    window.location.assign(`${loginUrl}?return=${returnUrl}`)
```

Comment:
I could have moved this code inside router.ts, but this is cleaner ;)

Reply:
onReady will get called after user authenticated. Hope its already handled and return URL will be dashboard

Reply:
How do you handle the 'bookmark' with this page?

Reply:
> onReady will get called after user authenticated. Hope its already handled and return URL will be dashboard

This page no longer includes the SbcSignin component, so there is no longer an `onReady` event handler. This page is just a redirect to the BC Registy login page, which performs the login and then redirects to the `return` URL specified here.

Reply:
> How do you handle the 'bookmark' with this page?

This page is transient and doesn't even display anything. The user may have bookmarked the "base" URL (which is the dashboard for this entity, eg, https://dev.bcregistry.ca/business/BC0870720), or they may bookmark the BC Registy login page (eg, https://yfthig-dev.web.app/login/), or perhaps a Business Registry link (marketing page or dashboard).

So if the user bookmarked the base URL and then are logged in, they just see the entity's dashboard. If they are not logged in then they will be redirected to the generic login page (BC Registry login) first.

Reply:
Let me know if this is still now clear. I could discuss it at the next Tech Demo.

Reply:
I mean if someone bookmark this page "https://dev.bcregistry.ca/business/BC0870720/standalone-directors", after the user login it will go to the main dashboard and won't go to director page. The business pages might be ok. It won't have lots of layer.

Reply:
Good example... but it will not work anyway because [this filing expects some app tombstone data](https://github.com/bcgov/business-filings-ui/blob/1c4f31189fad1ee53db895b818ed7dc63cf50592/src/views/StandaloneDirectorsFiling.vue#L468), which is very contextual (the app needs to have loaded all other entity data first). All filings are like this, so you cannot restore a bookmark to any filing. This is by design: we would not want a user to create an AR filing if one isn't due, etc.

Reply:
... And if you're wondering about the account ID URL parameter, that's [a different ticket](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10999) :)